barman/postgres.py

```diff
@@ -279,6 +279,10 @@ def server_major_version(self):
             return simplify_version(result)
         return None
 
+    def is_minimal_postgres_version(self):
+        """Checks if postgres version has at least minimal version"""
+        return self.server_version >= 90600
```

Comment:
Support for 9.6 was dropped in the 3.0.0 release so this should probable be `>= 100000`.

Reply:
I wanted to do it step by step to be sure I will not miss anything.